Lib/contextlib.py

```diff
@@ -191,6 +191,14 @@ class _AsyncGeneratorContextManager(
 ):
     """Helper for @asynccontextmanager decorator."""
 
+    def __call__(self, func):
+        @wraps(func)
+        async def inner(*args, **kwds):
+            async with self.__class__(self.func, self.args, self.kwds):
+                return await func(*args, **kwds)
+
+        return inner
+
```

Comment:
this function is now redundant - support for asynccontextmanagers as decorators was added in https://github.com/python/cpython/pull/20516